### PR TITLE
Task#7501

### PR DESCRIPTION
**Pull request by HeorhiiBoliuk**
Implement to taxonomy page fashion 3 components on the views. Do some changes in views, components.

### CODE REVIEW

**Comment by HeorhiiBoliuk** [comment]
This branch created from branch #21

File: web/themes/custom/my_awesome_theme/components/03-organisms/site/site-fashion-page/_site-fashion-page.scss
Comment (ivandudardb) [inline web/themes/custom/my_awesome_theme/components/03-organisms/site/site-fashion-page/_site-fashion-page.scss:1]: what is that?

**Review by ivandudardb — APPROVED** [review]
have empty file but anyway LGTM

File: web/themes/custom/my_awesome_theme/components/03-organisms/site/site-fashion-page/_site-fashion-page.scss
Comment (HeorhiiBoliuk) [reply web/themes/custom/my_awesome_theme/components/03-organisms/site/site-fashion-page/_site-fashion-page.scss:1]: File created for next task

Comment (lavrenchuk-dev) [reply web/themes/custom/my_awesome_theme/components/03-organisms/site/site-fashion-page/_site-fashion-page.scss:1]: So then it should be in next task :)

File: web/themes/custom/my_awesome_theme/components/03-organisms/site/site-gadgets-page/_site-gadgets-page.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
-/**.layout--onecol{
+
 @todo use this code for fashion taxonomy page
```
Comment (lavrenchuk-dev) [inline web/themes/custom/my_awesome_theme/components/03-organisms/site/site-gadgets-page/_site-gadgets-page.scss:2]: Is that scss even compiles
You removed start of the comment, left the end, and also this @todo seems outdated now.

**Review by lavrenchuk-dev — CHANGES_REQUESTED** [review]

**Review by lavrenchuk-dev — APPROVED** [review]
3/4 Commits have the wrong naming btw.

**Pull request merged**